lib/elastic_apm/grape.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'elastic_apm/subscriber'
+require 'elastic_apm/normalizers/grape'
+
+module ElasticAPM
+  # Module for starting the ElasticAPM agent and hooking into Grape.
+  module Grape
+    extend self
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    # Start the ElasticAPM agent and hook into Grape.
+    #
+    # @param app [Grape::API] A Grape app.
+    # @param config [Config, Hash] An instance of Config or a Hash config.
+    # @return [true, nil] true if the agent was started, nil otherwise.
+    def start(app, config)
```

Comment:
@mikker What do you think of this API? It centralizes the setting of config values into this file so it's not in the config (as we discussed).
However, it's inconsistent with `Rails.start` (which only accepts the `config` as a param)
I could also have it be `Grape.start(config, app: GrapeApp)`

What do you think?

Reply:
I think I like it best like this. We'll also eventually have to do the same for a `Sinatra` helper that will also include the app as an argument. I see your point about being inconsistent with Rails but I don't think it's a problem. Most people(™) using Rails will probably not start the agent themselves anyway?

Reply:
ok, I also like this API the best. I'll create a ticket to change the Rails API for 4.0. Good point about few people using `ElasitcAPM::Rails.start(..)`
I'll open a second PR to add a `ElasitcAPM::Sinatra.start(MyApp, config)` API, if you agree?

Reply:
Yes please 🙏